src/main/kotlin/apply/domain/applicant/ApplicantRepository.kt

```diff
@@ -0,0 +1,8 @@
+package apply.domain.applicant
+
+import org.springframework.data.jpa.repository.JpaRepository
+import java.util.Optional
+
+interface ApplicantRepository : JpaRepository<Applicant, Long> {
+    fun findByName(name: String): Optional<Applicant>
```

Comment:
`Optional<Applicant>` 대신 `Applicant?`를 사용하면 어떨까요?

```suggestion
    fun findByName(name: String): Applicant?
```